tutorials/translation/ct2_hindi_translation.ipynb

```diff
@@ -0,0 +1,623 @@
+    "class ModelForSeq2SeqModel(HFModel):\n",
+    "    def __init__(self, config):\n",
+    "        self.trans_config = config\n",
+    "        self.config = self.load_cfg()\n",
+    "        super().__init__(\n",
+    "            self.trans_config.pretrained_model_name_or_path, model_output_type=\"string\"\n",
+    "        )\n",
+    "\n",
+    "    def load_model(self, device=\"cuda\"):\n",
+    "        model = CT2CustomModel(self.trans_config)\n",
+    "        return model\n",
+    "\n",
+    "    def load_config(self):\n",
+    "        return self.load_cfg()\n",
+    "\n",
+    "    def load_tokenizer(self):\n",
+    "        return AutoTokenizer.from_pretrained(\n",
+    "            pretrained_model_name_or_path=self.trans_config.pretrained_model_name_or_path,\n",
+    "            trust_remote_code=True,\n",
+    "        )\n",
+    "\n",
+    "    def max_seq_length(self) -> int:\n",
+    "        return self.config.max_source_positions\n",
+    "\n",
+    "    def load_cfg(self):\n",
+    "        config = AutoConfig.from_pretrained(\n",
+    "            pretrained_model_name_or_path=self.trans_config.pretrained_model_name_or_path,\n",
+    "            trust_remote_code=True,\n",
+    "        )\n",
+    "        return config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "### Define IndicTranslation class\n",
+    "\n",
+    "Now that we have created relevent model classes from crossfit side, for running the pipeline we need to inherit from __DistributedDataClassifier__ of nemo-curator, and implement its **_run_classifier** method inside __IndicTranslation__\n",
+    "\n",
+    "_run_classifier method is responsible for running the inference. For our translation use case we need to have preprocessing, filtering before call for inference and postprocessing after the inference.\n",
+    "\n",
+    "In this example we have added pre and postprocessing from _run_classifier method. Overall major steps will be as follows : \n",
+    "\n",
+    "1.  Run process_input_text method which will be responsible for breaking english sentences via nltk's sentence tokenizer into sentences of specified length(default = 200 words).\n",
+    "2.  Filter data where sentence should at least have 1 alphabet in it.\n",
+    "3.  Left over data from step 2 won't go for translation but will be added for final data with translation as same as input text.\n",
+    "4.  Data which passed from step 2 will go fro indic preprocessing from IndicTransToolkit.\n",
+    "5.  CrossFit's Toeknizer and Predictor will run on data.\n",
+    "6.  Output from step 5 will go for detokenization and indic postprocessing.\n",
+    "7.  Combining the results from step 6 and step 3 and reutrn the data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class IndicTranslation(DistributedDataClassifier):\n",
```

Comment:
Same comment as above, why don't we move all of this logic into https://github.com/NVIDIA/NeMo-Curator/tree/main/nemo_curator/classifiers?